res/database/Default/user/scr0llbaer-constr-screenplays-2024-08.xml

```diff
@@ -0,0 +1,4279 @@
+			<targetgroupattractivity men="0.75" />
+			<genres maingenre="4" subgenres="7,17" />
+			<blocks value="2" />
+			<price min="150000" max="300000" />
+			<potential min="30" max="70" />
+			<outcome min="30" max="70" />
+			<review min="30" max="70" />
+			<speed min="30" max="70" />
+			<studio_size value="2" />
+		</scripttemplate>
+		<scripttemplate product="1" licence_type="1" guid="cff59d4b-cf24-4e87-97c9-dae487c996d6" creator="9551" comment="https://de.wikipedia.org/wiki/Till_Raether | https://www.goodreads.com/book/show/7646366-das-leben-ist-nur-eine-phase">
+			<title>
+				<de>Das Leben geht auch wieder vorbei</de>
+				<en>Life Is Just a Phase</en>
+			</title>
+			<title_original>
+				<de>Das Leben ist nur eine Phase</de>
+				<en>Life Is Just a Phase</en>
+			</title_original>
+			<description>
+				<de>Ein junger Mann hat sieben Tage frei und möchte sein Leben umkrempeln. Mal ausschlafen, dann Liste machen: Carrera-Bahn kaufen, Kinder kriegen... Die Angetraute ist davon etwas überrascht, und natürlich läuft nicht alles so nach Plan.</de>
+				<en>A young man has seven days off and wants to turn his life around. Sleep in, then make a list: buy a racetrack, have children... Wifey is a little surprised, and of course not everything goes according to plan.</en>
+			</description>
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" />
+				<job index="2" function="2" required="1" gender="2" />
+				<job index="3" function="2" required="1" gender="1" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="32" required="1" gender="1" />
+				<job index="6" function="32" required="1" gender="2" />
+			</jobs>
+			<data flags="0" flags_optional="" target_groups="6" />
+			<targetgroupattractivity men="0.75" />
+			<genres maingenre="15" subgenres="5,7" />
+			<blocks value="2" />
+			<price min="150000" max="300000" />
+			<potential min="30" max="70" />
+			<outcome min="30" max="70" />
+			<review min="30" max="70" />
+			<speed min="30" max="70" />
+			<studio_size value="2" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="ed95c50e-c70b-47db-9083-426fc3ed2757" creator="9551" comment="https://en.wikipedia.org/wiki/Charlie_Huston | https://en.wikipedia.org/wiki/Joe_Pitt_Casebooks | Terms like 'Vampyre', 'Vyrus' are [sic] with the 'y'.">
+			<title>
+				<de>Das %ROLE1% Dossier</de>
+				<en>The %ROLE1% Dossier</en>
+			</title>
+			<title_original>
+				<de>Die Joe Pitt Casebooks</de>
+				<en>The Joe Pitt Casebooks</en>
+			</title_original>
+			<description>
+				<de>%ROLE1%, ein „Vampyre”, schlägt sich im New Yorker Untergrund durch und erledigt im Tausch gegen Blut und Freiheit Aufträge für verschiedene Clans, während sich ein mysteriöser Zombie-„Vyrus“ ausbreitet.</de>
```

Comment:
"wrong" closing quote `”` in the beginning

Reply:
ack